prow/integ-suite-kind.sh

```diff
@@ -42,7 +42,7 @@ KIND_CONFIG=""
 CLUSTER_TOPOLOGY_CONFIG_FILE="${ROOT}/prow/config/topology/multicluster.json"
 
 export FAST_VM_BUILDS=true
-export ISTIO_DOCKER_BUILDER=crane
+export ISTIO_DOCKER_BUILDER=docker
```

Comment:
I guess this is the change that allows you have have `RUN` commands in the proxy dockerfile?

Reply:
Yes, you're right. As you can see, I updated the PR description and mentioned about this variable.